core/src/databases/remote_databases/snowflake.rs

```diff
@@ -136,6 +136,19 @@ impl SnowflakeRemoteDatabase {
             },
         )?;
 
+        if env::var("PROXY_HOST").is_ok()
```

Comment:
nit `if let` would allow to pattern match and extract the values without needing `is_ok` + `unwrap`

Reply:
Updating, thanks !

Reply:
Done.